qiita_pet/templates/sitebase.html

```diff
@@ -241,6 +243,11 @@
                 <li>
                   <a href="http://github.com/biocore/qiita">GitHub</a>
                 </li>
+                <li role="separator" class="divider"></li>
+                <li>
+                  <a href="#"><h6>{{qiita_version}}</h6></a>
```

Comment:
The version and SHA makes perfect sense to me, but not sure it will make sense to everyone, do you think it would help adding a "Current version: XXXXXXXX" label or something along those lines?